Update README.md

smyadari opened the pull request:
Updated all the links to use 'current'

smyadari committed:
Update README.md

Updated all the links to use 'current'

TeraDonawa commented on README.md:
```diff
@@ -231,19 +231,19 @@ Client. Key Sections of the guide include the System requirements,
 Installing, Configuring, Features, Troubleshooting.
 
 - [Anyware client 24.10 for Windows Administrators'
```
Remove the version number.  The text should change from something like "_Anyware client 24.10 for Windows Administrators' Guide_" "_Anyware client for Windows Administrators' Guide_"

TeraDonawa reviewed (changes requested):
The title references the explicit version of the guide, while the link points to the most recent version i.e. they're mismatched. We should adjust the title

TeraDonawa commented on README.md:
```diff
@@ -692,16 +692,16 @@ HP Anyware allows configuring the Anyware Agent to optimize the PCoIP
 Apart from the above hyperlinked guides, you can also utilize the
 session [performance optimization
-guide](https://anyware.hp.com/web-help/pcoip/session_planning_guide/2024.10/optimization/performance-optimization/)
+guide](https://anyware.hp.com/web-help/pcoip/session_planning_guide/current/optimization/performance-optimization/)
```
I also noticed on line 714 that the url help.teradici.com is in italics, whilst the other urls are in bold. Can you make it consistent (presumably all in bold)

smyadari committed:
Update README.md

1. Updated the text to remove version
2. Modified all the URLs from Italic format

TeraDonawa committed:
Update README.md
